tox.ini

```diff
@@ -12,11 +12,32 @@ extras =
     development
     pandas
 deps = pip
+external_wheels =
+    py37-ci: dist/*.whl
+    py38-ci: dist/*.whl
+    py39-ci: dist/.whl
+    py310-ci: dist/.whl
 passenv =
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    SF_PROJECT_ROOT
     cloud_provider
+    SF_REGRESS_LOGS
+    ; Github Actions provided environmental variables
     GITHUB_ACTIONS
+    JENKINS_HOME
+    ; This is required on windows. Otherwise pwd module won't be imported successfully,
+    ; see https://github.com/tox-dev/tox/issues/1455
+    USERNAME
+    CLIENT_LOG_DIR_PATH_DOCKER
     PYTEST_ADDOPTS
-setenv = COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
+setenv =
+    COVERAGE_FILE = {env:COVERAGE_FILE:{toxworkdir}/.coverage.{envname}}
+    ci: SNOWFLAKE_PYTEST_OPTS = -vvv
+    # Add common parts into pytest command
+    SNOWFLAKE_PYTEST_COV_LOCATION = {env:JUNIT_REPORT_DIR:{toxworkdir}}/junit.{envname}-{env:cloud_provider:dev}.xml
+    SNOWFLAKE_PYTEST_COV_CMD = --cov snowflake.sqlalchemy --junitxml {env:SNOWFLAKE_PYTEST_COV_LOCATION} --cov-report=
+    SNOWFLAKE_PYTEST_CMD = pytest {env:SNOWFLAKE_PYTEST_OPTS:} {env:SNOWFLAKE_PYTEST_COV_CMD}
```

Comment:
You added all these things, but you never use the final `SNOWFLAKE_PYTEST_CMD`

Reply:
removed unused env vars